### PR TITLE
Game Instruction

### PR DESCRIPTION
**Pull request by haraphat01**
This is the final submission for Tic Tac Toe on Ruby Curriculum.

What have we done in this fourth milestone?

- The Readme file has been updated

- The Readme file instructs any person on how to play the game.

- It also tells how the game was built, the necessary tools used.

- It also shows the author of the game.

The branch name has been changed and the readme has been updated with the game's U.I

### CODE REVIEW

**Review by stratospherique — CHANGES_REQUESTED** [review]
### Great work @team :clap: :clap: Almost done. However, some changes need to be applied in order to get approval for this project. Check :arrow_down: 

* [ ] It has descriptive branch name. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone4/branch_name.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))
  - Use a more specific name for this branch.
* [ ] README file looks professional. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone4/prof_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))
  - Add Screenshots of the UI of the game.
### That's it :relieved: When finished, kindly submit a code re-review request :pray: 

Happy coding :v: 
**_Ahmed Mahfoudh_**

File: README.md [hunk 1]
```diff
@@ -11,31 +11,46 @@ The project specifications can be seen in [Odin Project](https://www.theodinproj
...
+  <li>Get a valid user input</li>
+  <li>Display board after each move</li>
+  <li>End game if someone wins</li>
+  <li>End game if board becomes full</li>
+</ul>
 
 ## Built With
 
 - Ruby 2.6.5
 - VScode
```
Comment (santiago-dev96) [inline README.md]: Unnecessary, it doesn't matter what text editor you used.
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -11,31 +11,46 @@ The project specifications can be seen in [Odin Project](https://www.theodinproj
...
+- Rubocop (Linter) with Stickler (CI Tool)
```
Comment (santiago-dev96) [inline README.md]: Unnecessary.
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -11,31 +11,46 @@ The project specifications can be seen in [Odin Project](https://www.theodinproj
...
 - Git (with Gitflow),
```
Comment (santiago-dev96) [inline README.md]: Unnecessary.
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -11,31 +11,46 @@ The project specifications can be seen in [Odin Project](https://www.theodinproj
...
+
+You can play the live demo using this link: [Run on Repl.it](https://repl.it/@haraphat01/Tictactoegame#bin/main.rb)
+
 ## Quick Start
 
 ### Get a local copy with
 
 ```js
-git clone https://github.com/DickensJuma/tic_tac_toe_game
+git clone https://github.com/DickensJuma/Tic_tac_toe_game/pull/5
```
Comment (santiago-dev96) [inline README.md]: You don't clone the PR, you clone the repository.
```suggestion
git clone https://github.com/DickensJuma/Tic_tac_toe_game.git
```

File: README.md [hunk 5]
```diff
@@ -71,10 +86,14 @@ Ruby 2.5.5
...
 
-You can do it on [issues page](https://github.com/DickensJuma/tic_tac_toe_game/issues).
+You can do it on [issues page](https://github.com/DickensJuma/Tic_tac_toe_game/issues).
 
 ## Acknowledgments
 
+- [Odin Project](https://www.theodinproject.com/courses/ruby-programming/lessons/oop)
+- A special thanks for the code reviewers.
+## Acknowledgments
+
```
Comment (santiago-dev96) [inline README.md]: This content is repeated.
```suggestion
```

**Review by santiago-dev96 — CHANGES_REQUESTED** [review]
# Changes required :wrench:

Your `README.md` file is not professional yet. Also, it will be great if you could record a gif of the game being played and place it on the `README.md` file so the users get a better idea of how it is supposed to be played. On the other hand, the branch name **fourth-milestone** is not descriptive, maybe something like **feature/game-documentation** could work, I know it doesn't matter at the end, but I have to make sure that you are following professional best practices.

You can reach out to me in slack as **srodrig**, happy coding! :wave:

**Review by adaorachi — CHANGES_REQUESTED** [review]
### Hi @haraphat01 and @DickensJuma, 

Well done on the submission of this project. However, the branch for this pull request does not have any of the changes requested in the previous review. But I also noticed you have another branch with the name `readme_game_instructions`, and which seems to contain the right changes. 

With this, I guess you submitted the URL for this pull request from an oversight. The possible solutions to get your project properly reviewed and approved are:
- To change the name of this branch and include the right changes in it and request another code review with this same PR URL. [Here ](https://linuxize.com/post/how-to-rename-local-and-remote-git-branch/) is a detailed article in renaming a branch,   **OR**
- You create a pull request with the branch that has the correct changes and submit the URL to that Pull request in the code review form.

### **Status: Changes required ♻️** 

---
### **Kindly make any of the changes suggested above and submit another code review form.**

---

Should you have any question, please reach out to me on Slack with the username - @adaorachi

Happy Coding ☺️ 

**MaryAnn Chukwuka**

**Pull request closed without merging**